src/styles/main.scss

```diff
@@ -1 +1,61 @@
 // put code here
+@import "./utils/mixins.scss";
+
+$size-watch: 80vmin;
+$percent-value: 50%;
```

Comment:
Try to name variables not by content, but by meaning, because the content can change (for example, percentages will change on pixels) during development and you will have to fix it in the entire code.